easyocr/easyocr.py

```diff
@@ -423,8 +444,522 @@ def recognize(self, img_cv_grey, horizontal_list=None, free_list=None,\
+                         text_threshold = 0.7, low_text = 0.4, link_threshold = 0.4,\
+                         canvas_size = 2560, mag_ratio = 1.,\
+                         slope_ths = 0.1, ycenter_ths = 0.5, height_ths = 0.5,\
+                         width_ths = 0.5, y_ths = 0.5, x_ths = 1.0, add_margin = 0.1, 
+                         threshold = 0.2, bbox_min_score = 0.2, bbox_min_size = 3, max_candidates = 0,
+                         output_format='standard'):
+        '''
+        Parameters:
+        image: file path or numpy-array or a byte stream object
+        When sending a list of images, they all must of the same size,
+        the following parameters will automatically resize if they are not None
+        n_width: int, new width
+        n_height: int, new height
+        '''
+        img, img_cv_grey = reformat_input_batched(image, n_width, n_height)
+
+        horizontal_list_agg, free_list_agg = self.detect(img, 
+                                                 min_size = min_size, text_threshold = text_threshold,\
+                                                 low_text = low_text, link_threshold = link_threshold,\
+                                                 canvas_size = canvas_size, mag_ratio = mag_ratio,\
+                                                 slope_ths = slope_ths, ycenter_ths = ycenter_ths,\
+                                                 height_ths = height_ths, width_ths= width_ths,\
+                                                 add_margin = add_margin, reformat = False,\
+                                                 threshold = threshold, bbox_min_score = bbox_min_score,\
+                                                 bbox_min_size = bbox_min_size, max_candidates = max_candidates
+                                                 )
+        result_agg = []
+        # put img_cv_grey in a list if its a single img
+        img_cv_grey = [img_cv_grey] if len(img_cv_grey.shape) == 2 else img_cv_grey
+        for grey_img, horizontal_list, free_list in zip(img_cv_grey, horizontal_list_agg, free_list_agg):
+            result_agg.append(self.recognize(grey_img, horizontal_list, free_list,\
+                                            decoder, beamWidth, batch_size,\
+                                            workers, allowlist, blocklist, detail, rotation_info,\
+                                            paragraph, contrast_ths, adjust_contrast,\
+                                            filter_ths, y_ths, x_ths, False, output_format))
+
+        return result_agg
+
+
+
+
+class Custom_Reader(object):
+
+    def __init__(self, lang_list, opt=None, device=None, model_storage_directory=None,
+                 user_network_directory=None, detect_network="craft", download_enabled=True, 
+                 detector=True, recognizer=True, verbose=True, 
+                 quantize=True, cudnn_benchmark=False, model_path=None ,yaml_path=None, model_arch=None):
```

Comment:
we have added `model_path=None ,yaml_path=None, model_arch=None)` remove this parameters